client/a8c-for-agencies/sections/sites/sites-header-actions/index.tsx

```diff
@@ -48,13 +50,19 @@ export default function SitesHeaderActions( { onWPCOMImport }: Props ) {
 					onCreateSiteSuccess={ onCreateSiteSuccess }
 				/>
 			) }
-			<div ref={ ( ref ) => setTourStepRef( ref ) }>
-				<AddNewSiteButton
-					showMainButtonLabel={ ! isMobile }
-					onWPCOMImport={ onWPCOMImport }
-					toggleDevSiteConfigurationsModal={ toggleDevSiteConfigurationsModal }
-				/>
-			</div>
+			{ isEnabled( 'a4a-updated-add-new-site' ) ? (
+				<div ref={ ( ref ) => setTourStepRef( ref ) }>
+					<AddNewSite />
+				</div>
+			) : (
+				<div ref={ ( ref ) => setTourStepRef( ref ) }>
+					<AddNewSiteButton
+						showMainButtonLabel={ ! isMobile }
+						onWPCOMImport={ onWPCOMImport }
+						toggleDevSiteConfigurationsModal={ toggleDevSiteConfigurationsModal }
+					/>
+				</div>
+			) }
```

Comment:
At this point we could actually avoid the duplication and move the conditional logic inside the `div`, that way React will have one fewer element to treat:

```suggestion
			<div ref={ ( ref ) => setTourStepRef( ref ) }>
				{ isEnabled( 'a4a-updated-add-new-site' ) ? (
					<AddNewSite />
				) : (
					<AddNewSiteButton
						showMainButtonLabel={ ! isMobile }
						onWPCOMImport={ onWPCOMImport }
						toggleDevSiteConfigurationsModal={ toggleDevSiteConfigurationsModal }
					/>
				) }
			</div>
```

Reply:
Sorry, I overlooked this one. Fixed it now: https://github.com/Automattic/wp-calypso/pull/97499/commits/fca51b48b4cb37a59940787dcfce45e901850041